<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleAnalyzer</class>
 <widget class="QWidget" name="SingleAnalyzer">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Single Tweet Analyzer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="source.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#lbl_title {
font: 75 20pt &quot;Microsoft YaHei UI&quot;;
font-weight: bold;
qproperty-alignment: AlignCenter;
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #3643f7, stop:1 #3643f7);
}

#lbl_model {
font: 75 11pt &quot;Microsoft YaHei UI&quot;;
font-weight: bold;
color: black;
padding-left: 1px
}

QComboBox {
font: 75 10.5pt &quot;Microsoft YaHei UI&quot;;
font-weight: bold;
color: #1F1F21;
 border: 1px solid gray;
 border-radius: 10px;
 min-width: 6em;
padding-left: 8px;
qproperty-alignment: AlignCenter;
}
QComboBox::hover {
 border: 1px solid blue;
background: #f0f5f9;
}
/* QComboBox gets the &quot;on&quot; state when the popup is open */
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                                stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);
}
QComboBox:on {
    borde-radius: 10px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 25px;
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 10px; /* same radius as the QComboBox */
    border-bottom-right-radius: 10px;
}
QComboBox QAbstractItemView {
    border-radius: 10px;
    background: white;
    border: 1px solid gray;
    box-shadow: transparent;
    selection-background-color:lightblue;
	color: #1F1F21;
}

QComboBox::down-arrow {
image: url(:/images/arrow_down.png);
    width: 15px;
    height: 15px;
padding-right: 1px;
}

QTextEdit {
border: 1px solid; 
border-radius:10px;
background-color: palette(base);
font: 75 14pt &quot;Georgia, serif&quot;;
font-weight: bold;
color: #3f72af;
 min-width: 6em;
}

QTextEdit::disabled {
background-color: #D3D3D3;
}

#progressbar_batch::disabled {
background-color: #D3D3D3;
}
#lbl_result {
font: 75 18pt &quot;Microsoft YaHei UI&quot;;
font-weight: bold;
qproperty-alignment: AlignCenter;
color: #3f72af;
}

#btn_homepage {
background: transparent;
border-radius: 19px;
}

#btn_homepage::hover {
background: #3f72af;
}

#btn_help {
background: transparent;
border-radius: 19px;
}

#btn_help::hover {
background: #3f72af;
}

QPushButton {
font: 75 13pt &quot;Microsoft YaHei UI&quot;;
font-weight: bold;
border: 1px solid; 
border-radius:10px;
background-color: #f0f5f9;
}
QPushButton::disabled {
background-color: #D3D3D3;
}
QPushButton::hover {
background: #c9d6df;
}
QPushButton:pressed {
background: #3f72af;
}
QProgressBar:horizontal {
border: 1px solid gray;
border-radius: 3px;
background: white;
padding: 0px;
}
QProgressBar::chunk:horizontal {
background: qlineargradient(x1: 0, y1: 0,    x2: 0, y2: 1,
    stop: 0 #a8ff78, stop: 1 #78ffd6);
}

QProgressBar:vertical {
border: 1px solid gray;
border-radius: 3px;
background: white;
padding: 0px;
}
QProgressBar::chunk:vertical {
background: qlineargradient(x1: 0, y1: 0,    x2: 0, y2: 1,
    stop: 0 #a8ff78, stop: 1 #78ffd6);
}

QProgressBar::chunk:vertical::disabled {
background: qlineargradient(x1: 0, y1: 0,    x2: 0, y2: 1,
    stop: 0 #ED213A, stop: 1 #93291E);
}

QGroupBox {
font: 75 13pt &quot;Microsoft YaHei UI&quot;;
font-weight: bold;
color: #3f72af;
}

#lbl_human_result, #lbl_bot_result {
font: 75 13pt &quot;Microsoft YaHei UI&quot;;
font-weight: bold;
color: green;
}

#lbl_human_result, #lbl_bot_result::disabled {
color: red;
}

#pic_human {
image: url(:/images/human.png);
}

#pic_bot {
image: url(:/images/bot.png);
}


</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="btn_homepage">
       <property name="toolTip">
        <string>Back To HomePage</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="source.qrc">
         <normaloff>:/images/homepage.png</normaloff>:/images/homepage.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_help">
       <property name="toolTip">
        <string>Open Help Documentation</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="source.qrc">
         <normaloff>:/images/help.png</normaloff>:/images/help.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_title">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Single Tweet Analyzer</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>86</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>60</number>
     </property>
     <property name="topMargin">
      <number>19</number>
     </property>
     <property name="rightMargin">
      <number>60</number>
     </property>
     <item>
      <widget class="QComboBox" name="combobox_model">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>124</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>270</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentText">
        <string>Select Model</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <property name="minimumContentsLength">
        <number>150</number>
       </property>
       <item>
        <property name="text">
         <string>Select Model</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <property name="topMargin">
      <number>8</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignTop">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/images/open_quotes.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textbox_tweet">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>150</height>
          </size>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::WidgetWidth</enum>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Enter a Tweet Content...</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignBottom">
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/images/close_quotes.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="topMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QProgressBar" name="progressbar_batch">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>17</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>17</height>
          </size>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="btn_start">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>122</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Predict</string>
         </property>
         <property name="icon">
          <iconset resource="source.qrc">
           <normaloff>:/images/predict.png</normaloff>:/images/predict.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="horizontalFrame_2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupbox_human">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Human</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>25</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>25</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>40</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>40</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="pic_human">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbl_human_result">
               <property name="text">
                <string>0%</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QProgressBar" name="progressbar_human">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>13</width>
                 <height>100</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>13</width>
                 <height>100</height>
                </size>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="textVisible">
                <bool>false</bool>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="format">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupbox_bot">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Bot</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>25</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>25</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>40</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>40</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="pic_bot">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbl_bot_result">
               <property name="text">
                <string>0%</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QProgressBar" name="progressbar_bot">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>13</width>
                 <height>100</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>13</width>
                 <height>100</height>
                </size>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="textVisible">
                <bool>false</bool>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="format">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
